src/args.rs

```diff
@@ -15,7 +15,7 @@ ARGS:
 OPTIONS:
     --exec <COMMAND>          Change the execution command from the default.
                               This is run when you hit the Enter on a path
-                              The default command is \"type\" on Windows, or \"cat\" on other platforms.
+                              The default command is \"notepad\" on Windows, or \"cat\" on other platforms.
```

Comment:
I changed the default command with `"notepad"` because I couldn't invoke `"type"`. I'm not familiar with Windows, so it might be inappropriate to set `"notepad"` 🤷 However, I confirmed `"notepad"` actually worked well as I expected.


https://user-images.githubusercontent.com/13130705/120478834-3beaba80-c3e8-11eb-9665-083765a08638.mov